micro-ui/web/micro-ui-internals/packages/modules/pgr/src/components/timelineInstances/rejected.js

```
@@ -0,0 +1,55 @@
+    let actions =
+      nextActions &&
+      nextActions.map((action, index) => {
+        if (action && (action !== "COMMENT") ) {
+          if((action!== "REOPEN" || (action === "REOPEN" && (Date?.now() - complaintDetails?.service?.auditDetails?.lastModifiedTime) < ComplainMaxIdleTime)))
+          return (
+            <Link key={index} to={`/${window?.contextPath}/citizen/pgr/${action.toLowerCase()}/${serviceRequestId}`}>
+              <ActionLinks>{t(`CS_COMMON_${action}`)}</ActionLinks>
+            </Link>
+          );
+        }
+      });
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)